scene/3d/physics/vehicle_body_3d.cpp

```diff
@@ -542,7 +542,7 @@ void VehicleBody3D::_resolve_single_bilateral(PhysicsDirectBodyState3D *s, const
 	if (body2) {
 		rel_pos2 = pos2 - body2->get_global_transform().origin;
 	}
-	//this jacobian entry could be re-used for all iterations
+	// This jacobian entry could be reused for all iterations.
```

Comment:
```suggestion
	// This Jacobian entry could be reused for all iterations.
```
While we're here we can fix this as well